### ISSUE TITLE
mismatch between number of components in `series` and length of `component_mask`

### ISSUE THREAD

**Issue by lams0018** [unit8co/darts#1645]
Hi,

I'm trying out the TFT model using the example in this site https://unit8co.github.io/darts/examples/13-TFT-examples.html.

Had replaced the data with my own dataset but encountered this error "mismatch between number of components in `series` and length of `component_mask`" when trying to evaluate the model. Everything was workign fine until this step.

May i know how to access this mask_component, or perhaps the root cause for the error? Thanks.

**Comment by dennisbader** [comment]
Hi @lams0018, could you provide a minimum reproducible example?

**Comment by lams0018** [comment]
Thanks, i managed to figure it out. A silly mistake of forgetting to also input "val_past_covariates" into model.fit(), when i passed val_series in.

On a seperate question,  model.fit() produces "val_loss" during the training process. Is there any way to access this? Intending to plot the loss to view the change across the training epochs. Keras is able to do so with with a history["loss"] attribute in their models. Would there be anything similar that's provided by Darts?

**Comment by dennisbader** [comment]
Yes, there are multiple ways. Have a look at #1644.
